docs/content/management-decision-logs.md

```diff
@@ -253,6 +253,36 @@ to track **remove** vs **upsert** mask operations.
 }
 ```
 
+### Dropping decisions
+
+Drop rules filters all decisions, which evaluate to `true`, before logging them.
+
+This rule will drop all requests to the _allow_ rule in the _kafka_ package, that returned _true_:
+```ruby
+package system.log
+
+drop {
+  input.path = "kafka/allow"
+  input.result
+}
+```
```

Comment:
For consistent formatting on the website, let's use another language annotation: 

```
```live:drop_rule_example:module:read_only
package system.log

drop {
  input.path == "kafka/allow"
  input.result
}
```

☝️ Explicit unification (`=`) tends to confuse people, so let's use `==` where possible

Reply:
Amen!